### ISSUE TITLE
Dynamically Add Items

### ISSUE THREAD

**Issue by bettyeti** [ilikenwf/nestedSortable#29]
Any documentation or demos for dynamically adding items with working event handlers?  For instance adding a list item with delete?

**Comment by kthornbloom** [comment]
Also interested in this. I figured simply calling the function again would do the trick, but then I just get a bazillion "Uncaught TypeError: Cannot read property 'left' of undefined" errors as I try to drag.

**Comment by Claud** [comment]
Exemple:
```javascript
$('#addNode').click(function(){
				$('ol.sortable').append('<li id="item-' + (new Date()).getTime() + '"><div>Новыей элемент</div></li>');
				$('ol.sortable').nestedSortable('refresh');
			});
```

**Comment by kthornbloom** [comment]
Yes, that works. Would be nice if that were added into the documentation!

**Comment by phena109** [comment]
As far as I have used the plugin, we can simply insert the html and it will display fine (with some display glitch). No need to refresh.

(maybe we are talking about different version, I use the 2.0alpha)

**Comment by actorius** [comment]
**Claud**, not working you solution. Classes does not applyed until item is not be changed or moved.

**Comment by illmaren** [comment]
Change a bit (so that the menuItem will be added like in the example but it can´t be deleted...any tipps?
